bayes_optim/base.py

```diff
@@ -407,7 +407,9 @@ def step(self):
 
         self.tell(X, func_vals)
 
-    def ask(self, n_point=None):
+    def ask(self, n_point=None, seed = None):
```

Comment:
```suggestion
    def ask(self, n_point=None, seed=None):
```

Reply:
Sorry, Diederick.. This seems to be the code style from PEP8..

Reply:
Perhaps I am wrong about.. It would be good if you could also check the coding convention from PEP8 :)